src/coreclr/jit/codegenarm64.cpp

```diff
@@ -1823,17 +1711,15 @@ void CodeGen::genCaptureFuncletPrologEpilogInfo()
                 SP_to_FPLR_save_delta -= MAX_REG_ARG * REGSIZE_BYTES;
             }
```

Comment:
I believe this varArgs branch can be removed. It's purpose was to make sure that PSPSym of funclet in a varargs method is at the same offset as the main method. This is no longer needed.

Reply:
Same for the `lvaMonAcquired` code branch above...

Reply:
Yes, quite a bit of code in the funclet prologs/epilogs looks unnecessary now.